public/lib/column-defaults.js

```diff
@@ -311,6 +311,17 @@ const columnDefaults = [{
     isNew: true,
     isSortable: true,
     sortField: ['statusInPrint']
+},{
```

Comment:
Re-ordered so the commissionedLength can be next to the webWordCount (and so more easily comparable). But keeping webWordCount and printWordCount next to each other.

This is the inverse order of the Ophan screen (wordCount / commissionedLength), but hard to balance all the needs.